ee/cis/win-10/cis-policy-queries-WIP.yml

```diff
@@ -0,0 +1,283 @@
+apiVersion: v1
+kind: policy
+spec:
+  name: >
+    CIS - Ensure 'Configure enhanced anti-spoofing' is set to 'Enabled'
+  platforms: win10
+  platform: windows
+  description: |
+    This policy setting determines whether enhanced anti-spoofing is configured for devices which support it.
+  resolution: |
+    To establish the recommended configuration via GP, set the following UI path to Enabled:
+    'Computer Configuration\Policies\Administrative Templates\Windows Components\Biometrics\Facial Features\Configure enhanced anti-spoofing'
+    Note: This Group Policy path may not exist by default. It is provided by the Group Policy template Biometrics.admx/adml that is included with the Microsoft Windows 10 Release 1511 Administrative Templates (or newer).
+  query: |
+    SELECT 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\\SOFTWARE\\Policies\\Microsoft\\Biometrics\\FacialFeatures\EnhancedAntiSpoofing' AND data = 1);
```

Comment:
Totes, the reason I do it is because the benchmarks pdf uses a `:` for the key and a `\` for the path so I wanted to keep differentiating between the key and path.

Reply:
Fair. That's probably a good idea.